### PR TITLE
fixes for mongo 4.4

### PR DESCRIPTION
**Pull request by StpMax**
It should make possible connect to MongoDB v4.4.

### CODE REVIEW

**Comment by mindsdbadmin** [comment]
****CLA Assistant Lite bot**** All contributors have signed the CLA  ✍️

File: mindsdb/api/mongo/op_msg_responders/add_shard.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#     },
+#     "$clusterTime": {
+#         "clusterTime": 6876008835761307650,
+#         "signature": {
+#             "hash": 00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00,
+#             "keyId": 0
+#         }
+#     },
+#     "operationTime": 6876008835761307649
+# }
```
Comment (George3d6) [inline mindsdb/api/mongo/op_msg_responders/add_shard.py:29]: Is it worth it to keep these large bits of logic commented out ?

**Review by George3d6 — APPROVED** [review]
Not my area of expertise, so I can't opine on it much, if the tests pass go ahead and merge :)

File: mindsdb/api/mongo/op_msg_responders/add_shard.py (hunk 1 above)
Comment (StpMax) [reply mindsdb/api/mongo/op_msg_responders/add_shard.py:29]: yes, i think we should keep it, for a time at least. Im not sure all problems with v4.4 gone. That not easy get dump of real answer. So if we will have problems with this answer - will be much easier find reason.

**Pull request merged**